bats/tests/helpers/commands.bash

```diff
@@ -84,7 +84,8 @@ rdsudo() {
     rdshell sudo "$@"
 }
 spin() {
-    "$PATH_RESOURCES/$PLATFORM/bin/spin$EXE" "$@" | no_cr
+    # spin may call itself recursively, so make sure it calls the correct binary
+    PATH="$PATH_RESOURCES/$PLATFORM/bin:$PATH" "$PATH_RESOURCES/$PLATFORM/bin/spin$EXE" "$@" | no_cr
```

Comment:
Does this do the correct thing on Windows? `$PATH` would be set within the WSL distribution (I believe it's Debian), but it wouldn't be set for Windows when `spin.exe` is launched.

But I guess CI is passing, so it's probably fine?

Reply:
That's a good point. I'll open a separate GitHub issue for it. I think it works fine for BATS on CI right now, but is a potential source for confusion in the future.

FWIW, I copied the logic from `docker_exe`:

```bash
docker_exe() {
    # Add path to bundled credential helpers to the front of the PATH
    PATH="$PATH_RESOURCES/$PLATFORM/bin:$PATH" "$PATH_RESOURCES/$PLATFORM/bin/docker$EXE" "$@" | no_cr
}
```

I was also briefly wondering if it is really necessary to invoke the tool with the full path if we already stuff the same path to the front of `PATH`. But then I decided that it can't hurt, and leave it alone for now.